tensorflow/contrib/rnn/python/kernel_tests/rnn_cell_test.py

```diff
@@ -1640,5 +1640,42 @@ def cell():
     self.assertAllClose(expected_c, actual_c, 1e-5)
     self.assertAllClose(expected_h, actual_h, 1e-5)
 
+
+class TestNLSTM(test.TestCase):
+
+  def test_nlstm_tuple(self):
```

Comment:
Please use camelCase, to be consistent in style with other parts of this file.

Reply:
Okay, will do.